### PR TITLE
Dotnet reference app

### PR DESCRIPTION
**Pull request by kamarthiparamesh**

### CODE REVIEW

File: samples/apps.json [hunk 1]
```diff
@@ -11,6 +11,11 @@
             "webOriginUrl": "http://localhost:8000"
         }
     },
+    "affinidi-dotnet-openidconnect": {
+        "redirectUris": {
+            "callbackUrl": "http://localhost:5068/signin-oidc"
```
Comment (pulkitb2) [inline samples/apps.json:16]: Can we keep the callback url as /callback

File: generator/dotnet/template/Readme.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+- [Visual Studio code](https://code.visualstudio.com/) (or any preferred code editor)
+
+    #### PreRequisite check 
+
+    Run this command to get the Dotnet version 
+    ```sh 
+    dotnet --version 
+    ```
+
+    You should get Output as, please reinstal if dotet version is not 8.0.xxx
```
Comment (pulkitb2) [inline generator/dotnet/template/Readme.md:23]: please change to reinstall and dotnet. You can setup a spelling check to find others as well

File: generator/dotnet/template/appsettings.Development.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*"
```
Comment (pulkitb2) [inline generator/dotnet/template/appsettings.Development.json:8]: Can this be made more secure? Allowing only particular hosts which interact with the app

File: generator/dotnet/template/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*"
```
Comment (pulkitb2) [inline generator/dotnet/template/appsettings.json:8]: same here

File: generator/dotnet/template/Readme.md (hunk 2 above)
Comment (sanjay95) [reply generator/dotnet/template/Readme.md:23]: changed

File: generator/dotnet/template/appsettings.json (hunk 4 above)
Comment (sanjay95) [reply generator/dotnet/template/appsettings.json:8]: restricted to localhost

File: generator/dotnet/template/appsettings.Development.json (hunk 3 above)
Comment (sanjay95) [reply generator/dotnet/template/appsettings.Development.json:8]: restricted to localhost

File: samples/apps.json (hunk 1 above)
Comment (sanjayK-aff) [reply samples/apps.json:16]: This I will take as next step as OIDC library uses this as default.

**Review by pulkitb2 — APPROVED** [review]

**Review by pulkitb2 — APPROVED** [review]

**Review by ivans-affinidi — APPROVED** [review]
Looks good.

**Review by carlos-affinidi — APPROVED** [review]

**Pull request merged**